Examples/Algorithms/TrackFinding/src/SeedingAlgorithm.cpp

```diff
@@ -205,6 +205,11 @@ ActsExamples::SeedingAlgorithm::SeedingAlgorithm(
         });
   }
 
+  m_cfg.seedFinderConfig.experimentCuts.connect(
```

Comment:
As discussed offline, this can be removed entirely.

Reply:
This was completely removed since it was implemented in Athena by Noemi in MR https://gitlab.cern.ch/atlas/athena/-/merge_requests/64366